src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveV3Ethereum_AaveFundingUpdates_20231106.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {AaveSwapper} from 'aave-helpers/swaps/AaveSwapper.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title Aave Funding Updates
+ * @author efecarranza.eth
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x099f88e1728760952be26fcb8fc99b26c29336e6a109820b391751b108399ee5
+ * - Discussion: https://governance.aave.com/t/arfc-aave-funding-update/15194
+ */
+contract AaveV3Ethereum_AaveFundingUpdates_20231106 is IProposalGenericExecutor {
+  using SafeERC20 for IERC20;
+
+  AaveSwapper public constant SWAPPER = AaveSwapper(MiscEthereum.AAVE_SWAPPER);
+
+  uint256 public constant USDC_TO_SWAP = 200_000e6;
+  uint256 public constant USDC_TO_DEPOSIT = 1_500_000e6;
+  uint256 public constant USDT_TO_DEPOSIT = 750_000e6;
+  uint256 public constant DAI_TO_SWAP = 500_000e18;
+  uint256 public constant USDC_TO_MIGRATE = 900_000e6;
+
+  address public constant MILKMAN = 0x11C76AD590ABDFFCD980afEC9ad951B160F02797;
+  address public constant PRICE_CHECKER = 0xe80a1C615F75AFF7Ed8F08c9F21f9d00982D666c;
+
+  // https://etherscan.io/address/0x3f12643D3f6f874d39C2a4c9f2Cd6f2DbAC877FC
+  address public constant GHO_ORACLE = 0x3f12643D3f6f874d39C2a4c9f2Cd6f2DbAC877FC;
+
+  function execute() external {
+    AaveV2Ethereum.COLLECTOR.transfer(
+      AaveV2EthereumAssets.USDC_A_TOKEN,
+      address(this),
+      USDC_TO_MIGRATE
+    );
+
+    AaveV2Ethereum.POOL.withdraw(
+      AaveV2EthereumAssets.USDC_UNDERLYING,
+      USDC_TO_MIGRATE,
```

Comment:
Instead of using the constant `USDC_TO_MIGRATE` I would use `type(uint256).max` to handle the aToken imprecision after transfer